content/en/docs/tasks/observability/distributed-tracing/overview/index.md

```diff
@@ -12,60 +12,48 @@ test: n/a
 Distributed tracing enables users to track a request through mesh that is distributed across multiple services.
 This allows a deeper understanding about request latency, serialization and parallelism via visualization.
 
-Istio leverages [Envoy's distributed tracing](https://www.envoyproxy.io/docs/envoy/v1.12.0/intro/arch_overview/observability/tracing) feature to provide tracing integration out of the box.
-Specifically, Istio provides options to install various tracing backends and configure proxies to send trace spans to them automatically. See [Zipkin](/docs/tasks/observability/distributed-tracing/zipkin/), [Jaeger](/docs/tasks/observability/distributed-tracing/jaeger/), and [Lightstep](/docs/tasks/observability/distributed-tracing/lightstep/) task docs about how Istio works with those tracing systems.
+Istio leverages [Envoy's distributed tracing](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing) feature to provide tracing integration out of the box.
 
-## Trace context propagation
+Most tracing backends now use the [OpenTelemetry](/docs/tasks/observability/distributed-tracing/opentelemetry/) protocol to receive traces, though Istio also supports legacy protocols for projects like [Zipkin](/docs/tasks/observability/distributed-tracing/zipkin/) and [Apache SkyWalking](/docs/tasks/observability/distributed-tracing/skywalking/).
```

Comment:
```suggestion
Most tracing backends now accept [OpenTelemetry](/docs/tasks/observability/distributed-tracing/opentelemetry/) protocol to receive traces, though Istio also supports legacy protocols for projects like [Zipkin](/docs/tasks/observability/distributed-tracing/zipkin/) and [Apache SkyWalking](/docs/tasks/observability/distributed-tracing/skywalking/).
```

Nit: I think `accept` here is more accurate, because backends accept OTLP but they may use their own formats internally.